src/main/kotlin/gg/skytils/skytilsmod/features/impl/handlers/Waypoints.kt

```diff
@@ -370,3 +383,32 @@ data class Waypoint @OptIn(ExperimentalSerializationApi::class) constructor(
         UGraphics.enableDepth()
     }
 }
+
+@Serializable
+data class Coleweight(
+    val x: Int,
+    val y: Int,
+    val z: Int,
+    val r: Int,
+    val g: Int,
+    val b: Int,
+    val options: ColeweightOptions
+) {
+    fun toSkytilsWaypoint(): Waypoint {
+        val color = Color(r * 255, g * 255, b * 255) // We assume that 0 is 0 and 1 is 255
```

Comment:
this doesn't really make sense to me if r, g, b are ints

Reply:
ig JSON.stringify parses 0.0 and 1.0 to 0 and 1 respectively? Looking at their code they are supposed to be floats ig

Reply:
> ig JSON.stringify parses 0.0 and 1.0 to 0 and 1 respectively? Looking at their code they are supposed to be floats ig (in case I am wrong I added a coerceIn)

In JavaScript all numbers are basically `double` types